tests/content/entity/zone/create/test.js

```diff
@@ -2,7 +2,7 @@ if (typeof PATH_TO_THE_REPO_PATH_UTILS_FILE === 'undefined') PATH_TO_THE_REPO_PA
 Script.include(PATH_TO_THE_REPO_PATH_UTILS_FILE);
 var autoTester = createAutoTester(Script.resolvePath("."));
 
-autoTester.perform("Zone create", Script.resolvePath("."), "primary", function(testType) {
+autoTester.perform("Zone create", Script.resolvePath("."), "secondary", function(testType) {
     // Enabled draw zone bounding box and stack to visualize the stack of zone components
     Render.getConfig("RenderMainView.DrawZoneStack").enabled = true;
     Render.getConfig("RenderMainView.DrawZones").enabled = true;
```

Comment:
If we're not rendering the zone stack anyways, we don't need to turn it on.

Reply:
Fixed